examples/nidigital_spi/README.md

```diff
@@ -25,5 +25,12 @@ This is a MeasurementLink example that tests an SPI device using an NI Digital P
 
 This example requires an NI Digital Pattern instrument (e.g. PXIe-6570).
 
-By default, this example uses a simulated instrument. To use a physical instrument, edit
-`_constants.py` to specify `USE_SIMULATION = False`.
\ No newline at end of file
+By default, this example uses a physical instrument or a simulated device created in NI MAX. To automatically create a simulated device when running the measurement or TestStand sequence, follow the steps below:
```

Comment:
Sorry to rewrite this again, but "create a simulated device" implies that it adds the device to NI MAX, but it doesn't. Also, we're mixing "instrument" and "device" here.

```suggestion
By default, this example uses a physical instrument or a simulated instrument created in NI MAX. To automatically simulate an instrument without using NI MAX, follow the steps below:
```

Reply:
> This applies to all of these files, including nidcpower_source_dc_voltage.

I'll fix this